context/app/api/vitessce_confs/assay_confs.py

```diff
@@ -0,0 +1,203 @@
+                    img_path, AssetPaths.IMAGE_PYRAMID_DIR.value
+                )
+                image_wrappers += [
+                    OmeTiffWrapper(
+                        img_url=img_url,
+                        offsets_url=offsets_url,
+                        name=self._get_hybcycle(img_path),
+                    )
+                ]
+            dataset = dataset.add_object(MultiImageWrapper(image_wrappers))
+            vc = self._setup_view_config_raster(vc, dataset)
+            conf = vc.to_dict()
+            # Don't want to render all layers
+            del conf["datasets"][0]["files"][0]["options"]["renderLayers"]
+            confs.append(conf)
+        self.conf = confs
+        return self
+
+    def _get_hybcycle(self, image_path):
+        return re.search(AssetPaths.SEQFISH_HYB_CYCLE_REGEX.value, image_path)[0]
+
+    def _get_pos_name(self, image_path):
+        return re.search(AssetPaths.SEQFISH_FILE_REGEX.value, image_path)[0].split(".")[
+            0
+        ]
+
+
+class CytokitSPRMConf(SPRMViewConf):
+    def __init__(self, entity, nexus_token, is_mock):
+        # All "file" Vitessce objects that do not have wrappers.
+        super().__init__(entity, nexus_token, is_mock)
+        self._files = [
+            {
+                "rel_path": f"{AssetPaths.CODDEX_SPRM_DIR.value}/"
+                + f"{AssetPaths.TILE_REGEX.value}.cells.json",
+                "file_type": ft.CELLS_JSON,
+                "data_type": dt.CELLS,
+            },
+            {
+                "rel_path": f"{AssetPaths.CODDEX_SPRM_DIR.value}/"
+                + f"{AssetPaths.TILE_REGEX.value}.cell-sets.json",
+                "file_type": ft.CELL_SETS_JSON,
+                "data_type": dt.CELL_SETS,
+            },
+            {
+                "rel_path": f"{AssetPaths.CODDEX_SPRM_DIR.value}/"
+                + f"{AssetPaths.TILE_REGEX.value}.clusters.json",
+                "file_type": "clusters.json",
+                "data_type": dt.EXPRESSION_MATRIX,
+            },
+        ]
+
+    def build_vitessce_conf(self):
+        file_paths_found = [file["rel_path"] for file in self._entity["files"]]
+        found_tiles = _get_matches(file_paths_found, AssetPaths.TILE_REGEX.value)
+        confs = []
+        for index, tile in enumerate(sorted(found_tiles)):
+            vc = VitessceConfig(name=tile)
+            dataset = vc.add_dataset(name="Cytokit + SPRM")
+            img_url, offsets_url = self._get_img_and_offset_url(
+                f"{AssetPaths.CODEX_TILE_DIR.value}/{tile}.ome.tiff",
+                AssetPaths.CODEX_TILE_DIR.value,
+            )
+            image_wrapper = OmeTiffWrapper(
+                img_url=img_url, offsets_url=offsets_url, name=tile
+            )
+            dataset = dataset.add_object(image_wrapper)
+            # This tile has no segmentations
+            if (
+                self._files[0]["rel_path"].replace(AssetPaths.TILE_REGEX.value, tile)
```

Comment:
This is a straight string replacement because this string is a regex to begin with: 
https://github.com/hubmapconsortium/portal-ui/pull/1581/files#diff-84f44cdfe4a5ffcaa8d5116f546a30b0a2c5a800753f39bdd69e050d5488546fR84-R85